gsdk/src/lib.rs

```diff
@@ -84,6 +84,24 @@ impl Api {
     }
 
     /// Subscribe all blocks
+    ///
+    /// # Parse events from blocks
+    ///
+    /// ```ignore
+    /// use gsdk::metadata::Event;
+    ///
+    /// let api = Api::new(None).await?;
+    /// let mut blocks = api.blocks().await?;
+    ///
+    /// while let Some(Ok(events)) = blocks.next_events().await {
```

Comment:
btw, can't we cast events as root events just like here?

Reply:
do you mean adding a new method `events`?

Reply:
#2896